src/darwin/Framework/CHIP/CHIPSetupPayload_Internal.h

```diff
@@ -17,7 +17,8 @@
 
 #ifdef __cplusplus
 - (id)initWithSetupPayload:(chip::SetupPayload)setupPayload;
-- (CHIPRendezvousInformationFlags)valueOf:(chip::RendezvousInformationFlags)value;
+- (CHIPRendezvousInformationFlags)convertRendezvousFlags:(chip::RendezvousInformationFlags)value
+- (CHIPCommissioningFlow)convertCommissioningFlow:(chip::CommissioningFlow)value
```

Comment:
```suggestion
- (CHIPRendezvousInformationFlags)convertRendezvousFlags:(chip::RendezvousInformationFlags)value;
- (CHIPCommissioningFlow)convertCommissioningFlow:(chip::CommissioningFlow)value;
```
might help the Darwin build.